tools/release-ci/cmd/cloneref.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/release/tools/release-ci/pkg/pullrefs"
+)
+
+// clonerefCmd clones a PULL_REFS specification
+var clonerefCmd = &cobra.Command{
+	Use:   "cloneref REPOSITORY_URL [PULL_REFS]",
+	Short: "Clones a PULL_REFS specification from Prow into the specified directory",
+	Long: `Clones a PULL_REFS specification from Prow into the specified directory
+	
+PULL_REFS is expected to be in the form of:
+
+base_branch:commit_sha_of_base_branch,pull_request_no:commit_sha_of_pull_request_no,...
```

Comment:
Upstream allows branch name not to have SHA -- we need to determine if we are going to interpret "build this with latest Origin RPMs" by build-time resolving the current stable SHA for Origin artifacts to get that info into this